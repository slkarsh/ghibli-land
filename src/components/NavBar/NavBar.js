import React from 'react';
import { Link } from 'react-router-dom'
import './NavBar.scss'


export const NavBar = () => {
  return (
  <nav className='nav-bar'>
    <ul className='ul-nav-list'>
      <Link to='/movies'>
        <li className='nav-list-item'>Movies</li>
      </Link>
      <Link to='/characters'>
        <li className='nav-list-item'>Characters</li> 
      </Link>
      <Link to='/places'>
        <li className='nav-list-item'>Places</li>
      </Link>
      <Link to='/vehicles'>
        <li className='nav-list-item'>Vehicles</li>
      </Link>
      <Link to='/design'>
        <li className='nav-list-item'>Design Your Own</li>
      </Link>
    </ul>
  </nav>
  )
}

export default NavBar