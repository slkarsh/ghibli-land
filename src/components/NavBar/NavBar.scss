nav {

}

.ul-nav-list {
  list-style: none;
  display: flex;
  justify-content: space-around;
}

.nav-list-item {

}