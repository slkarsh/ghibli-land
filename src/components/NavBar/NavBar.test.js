import { NavBar } from './NavBar'
import { shallow } from 'enzyme'
import React from 'react'

describe('NavBar', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<NavBar />)

    expect(wrapper).toMatchSnapshot()
  })
})