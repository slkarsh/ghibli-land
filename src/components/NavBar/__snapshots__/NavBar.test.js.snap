// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should match the snapshot 1`] = `
<nav
  className="nav-bar"
>
  <ul
    className="ul-nav-list"
  >
    <Link
      to="/movies"
    >
      <li
        className="nav-list-item"
      >
        Movies
      </li>
    </Link>
    <Link
      to="/characters"
    >
      <li
        className="nav-list-item"
      >
        Characters
      </li>
    </Link>
    <Link
      to="/places"
    >
      <li
        className="nav-list-item"
      >
        Places
      </li>
    </Link>
    <Link
      to="/vehicles"
    >
      <li
        className="nav-list-item"
      >
        Vehicles
      </li>
    </Link>
    <Link
      to="/design"
    >
      <li
        className="nav-list-item"
      >
        Design Your Own
      </li>
    </Link>
  </ul>
</nav>
`;
